setup.cfg

```diff
@@ -43,6 +43,11 @@ docs =
     nbsphinx
     sphinx-rtd-theme
 
+[options.entry_points]
+glue.plugins =
+    table_viewer = glue_jupyter.table:setup
```

Comment:
This doesn't exist, right? (the ``setup`` function). I agree with what you said in your other comments about not doing this yet for build-in viewers, so maybe remove for now.